boards/build.rs

```diff
@@ -0,0 +1,37 @@
+// Licensed under the Apache License, Version 2.0 or the MIT License.
+// SPDX-License-Identifier: Apache-2.0 OR MIT
+// Copyright Tock Contributors 2024.
+
```

Comment:
Please add a header doc comment with some of the info from the PR (what this file is and how a board should use it).